tests/test_SplicingBamFile.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+import os
+
+from epivizFileParser import SplicingBamFile
+
+__author__ = "jkanche, alexanv8"
+__copyright__ = "jkanche"
+__license__ = "mit"
+
+"""
+    The file test.bigBed and test assertions
+    come from the pyBigWig library
+"""
+
+bb = SplicingBamFile("tests/data/test.bam")
+
+
+def test_getRange():
+    (coverage, junctions), error = bb.getRange("chr10", 1, 100000000)
+    assert(error == None)
+
+    assert(len(bb.columns) == 6)
+    assert(bb.columns == ["chr", "region1_start", "region2_start",
+                          "region1_end", "region2_end", "value"])
+    assert(len(coverage) == 8)
+    assert(len(junctions) == 14)
```

Comment:
should test that start/end values equal expected values

Reply:
assertions added for value, start and end